src/styles/theming/sap_fiori_3.scss

```diff
@@ -264,6 +264,14 @@
   /* Link */
   --fdLink_Text_Decoration_Opposite: underline;
   --fdLink_Text_Decoration: none;
+  --fdLink_Text_Focus: var(--sapLinkColor);
```

Comment:
It would be preferable to move this out each theming file and into its own `src/styles/theming/common/link` directory to avoid duplication - but of course this needs to be done for a few other components, so I am ok allowing this PR and maybe the refactor can be handled for every component in a new PR

Reply:
Probably should be named `--fdLink_Text_Focus_Color`.

Reply:
@platon-rov renamed